driver-core/src/main/java/com/datastax/driver/core/Message.java

```diff
@@ -350,10 +363,10 @@ EnumSet<Frame.Header.Flag> computeFlags(Request request) {
       return flags;
     }
 
-    int encodedSize(Request request) {
+    int encodedSize(Request request, ProtocolFeatureStore featureStore) {
```

Comment:
It is already have ProtocolFeatureStore instance in it, use that one.
```suggestion
    int encodedSize(Request request) {
```